src/v3/src/transformer/messages/transform.ts

```diff
@@ -115,14 +117,19 @@ const transformBiometricsErrorMessage: TransformStepFnWithOptions = ({
   if (
     !transaction.messages
     || transaction.messages.length === 0
-    || !containsMessageKey(OV_UV_ENABLE_BIOMETRIC_SERVER_KEY, transaction.messages)
+    || !containsOneOfMessageKeys([
+      OV_UV_ENABLE_BIOMETRIC_SERVER_KEY,
+      OV_UV_RESEND_ENABLE_BIOMETRIC_SERVER_KEY,
```

Comment:
can you extract this array and the same one below to a variable?  Just so its a little more simple to add more biometric error keys in the future if needed